auth/jwt/README.md

```diff
@@ -0,0 +1,123 @@
+# package auth/jwt
+
+`package auth/jwt` provides a set of interfaces for service authorization through [JSON Web Tokens](https://jwt.io/).
+
+## Usage
+
+NewParser takes a key function and an expected signing method and returns an `endpoint.Middleware`. 
+The middleware will parse a token passed into the context via the `jwt.JWTTokenContextKey`. 
+If the token is valid, any claims will be added to the context via the `jwt.JWTClaimsContextKey`.
+
+```go
+import (
+	stdjwt "github.com/dgrijalva/jwt-go"
+    
+    "github.com/go-kit/kit/auth/jwt"
+    "github.com/go-kit/kit/endpoint"
+)
+
+func main() {
+	var exampleEndpoint endpoint.Endpoint
+	{
+		keyFunc := func(token *stdjwt.Token) (interface{}, error) { return []byte("SigningString"), nil }
+		jwtParser := jwt.NewParser(keyFunc, stdjwt.SigningMethodHS256)
```

Comment:
If possible, it would be great to avoid the necessity of importing 2 jwt packages to use this in the common/default case. Unfortunately I don't really know enough about jwt to make strong recommendations :) Is there any way to produce a standard keyFunc, or will every user need to do something totally custom? Is stdjwt.SigningMethodHS256 "preferred" enough that it's worth treating it as a default, and having either another constructor or a functional option to specify something different?